odxtools/cli/_print_utils.py

```diff
@@ -4,7 +4,8 @@
 from typing import Any, Callable, Dict, List, Optional, Union
 
 import markdownify
-from tabulate import tabulate  # TODO: switch to rich tables
```

Comment:
tabulate dependency not removed from pyproject.toml

Reply:
let me update that.

Reply:
I cannot remove tabulate from dependency some other files are using it. This PR was specifically for list command output.